cypress/integration/user_profile/UploadUserProfileHeader.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Upload UserProfile Header

```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-06-18T08:39:28Z" title="Thursday, June 18th 2020, 10:39:28 am +02:00">Jun 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+Feature: Upload UserProfile Header
```


Hey, very good you add this e2e test! 🤗😍